bin/agent-data-plane/src/cli/run.rs

```diff
@@ -173,6 +185,7 @@ async fn create_topology(
         .add_encoder("dd_events_encode", dd_events_config)?
         .add_encoder("dd_service_checks_encode", dd_service_checks_config)?
         .add_forwarder("dd_out", dd_forwarder_config)?
+        .add_destination("dsd_stats_out", dsd_stats_config.clone())?
```

Comment:
`dogstatsd-stats` isn't enabled by default in `datadog-agent` since there is a performance cost I believe. It is gated behind the `dogstatsd_metrics_stats_enable` env var.